src/components/Main/Results/DeterministicLinePlot.tsx

```diff
@@ -230,7 +239,11 @@ export function DeterministicLinePlot({ data, userResult, logScale, showHumanize
                   tickCount={7}
                 />
                 <YAxis scale={logScaleString} type="number" domain={[1, 'dataMax']} tickFormatter={yTickFormatter} />
-                <Tooltip formatter={tooltipFormatter} labelFormatter={labelFormatter} />
+                <Tooltip 
+                  formatter={tooltipFormatter} 
```

Comment:
You need to pass this tooltipFormatter to your ResponsiveTooltipContent and format the right hand value as necessary.  I don't think the tooltip is using the formatter, since custom content takes priority.